src/__tests__/ui/box/chrome.test.jsx

```diff
@@ -55,34 +55,6 @@ describe('Chrome', () => {
     expectComponent(<Chrome {...defaultProps} />).toMatchSnapshot();
   });
 
-  it('correctly sets the header size when the "signin ready" event is triggered', () => {
```

Comment:
why are these two removed now?

Reply:
Lock currently uses a dynamically-calculated header heigh for various reasons - mainly to fix weird layout issues - but those issues are resolved with this PR so that dynamic header height calculation has been removed, and along with it, these tests.